.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,359 @@
+        key: ${{ hashFiles('.ci/setup_env_github.sh') }}-${{ hashFiles('.github/workflows/build_and_test.yml') }}-${{ hashFiles('.requirements') }}
+
+    - name: Checkout kong-build-tools
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+      uses: actions/checkout@v2
+      with:
+        repository: Kong/kong-build-tools
+        path: kong-build-tools
+        ref: next
+
+    - name: Checkout go-pluginserver
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+      uses: actions/checkout@v2
+      with:
+        repository: Kong/go-pluginserver
+        path: go-pluginserver
+
+    - name: Add to Path
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+      run: echo "::add-path::$INSTALL_ROOT/openssl/bin:$INSTALL_ROOT/openresty/nginx/sbin:$INSTALL_ROOT/openresty/bin:$INSTALL_ROOT/luarocks/bin:$GITHUB_WORKSPACE/kong-build-tools/openresty-build-tools"
+
+    - name: Install packages
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+      run: sudo apt update && sudo apt install libyaml-dev
+
+    - name: Build Kong dependencies
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+      run: |
+          source .ci/setup_env_github.sh
+          make dev
+
+  lint-and-unit-tests:
+    name: Lint & Unit tests
+    runs-on: ubuntu-18.04
+    needs: build
+
+    env:
+      KONG_TEST_PG_DATABASE: kong
+      KONG_TEST_PG_USER: postgres
+
+    services:
+      postgres:
+        image: postgres:10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: kong
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 5s --health-timeout 5s --health-retries 8
+
+    steps:
+    - name: Set environment variables
+      run: |
+          echo "::set-env name=INSTALL_ROOT::$HOME/install-root"
+          echo "::set-env name=DOWNLOAD_ROOT::$HOME/download-root"
+          echo "::set-env name=LD_LIBRARY_PATH::$HOME/install-root/openssl/lib:$LD_LIBRARY_PATH"
+
+    - name: Checkout Kong source code
+      uses: actions/checkout@v2
+
+    - name: Lookup build cache
+      uses: actions/cache@v1
+      id: cache-deps
+      with:
+        path: ${{ env.INSTALL_ROOT }}
+        key: ${{ hashFiles('.ci/setup_env_github.sh') }}-${{ hashFiles('.github/workflows/build_and_test.yml') }}-${{ hashFiles('.requirements') }}
+
+    - name: Add to Path
+      run: echo "::add-path::$INSTALL_ROOT/openssl/bin:$INSTALL_ROOT/openresty/nginx/sbin:$INSTALL_ROOT/openresty/bin:$INSTALL_ROOT/luarocks/bin"
+
+    - name: Lint Lua code
+      run: |
+          eval `luarocks path`
+          luacheck -q .
+
+    - name: Check admin API documentation
+      run: |
+          eval `luarocks path`
+          scripts/autodoc-admin-api
+
+    - name: Unit tests
+      run: |
+          eval `luarocks path`
+          bin/busted -v -o gtest spec/01-unit
+
+  integration-tests-postgres:
+    name: Integration tests with PostgreSQL (${{ matrix.suite }} - ${{ matrix.split }})
```

Comment:
These long names are truncated so the interesting part is missing:

<img width="323" alt="Screenshot 2020-02-05 at 16 42 29" src="https://user-images.githubusercontent.com/63131/73857137-c6dc9480-4836-11ea-93f0-fc359ee6cc0e.png">

A simple reorder would work:

```suggestion
    name: PostgreSQL (${{ matrix.suite }} - ${{ matrix.split }}) Integration tests
```

Alternatively the names could be shorter:
```suggestion
    name: Integration Pg (${{ matrix.suite }}-${{ matrix.split }})
```

This applies to the other names on this file as well.

Reply:
Yes, please! I noticed the same thing.